test/e2e/ginkgo/ginkgo_suite_test.go

```diff
@@ -48,11 +48,21 @@ var (
 	its           *ocmWorkClient.Clientset
 	releasedFlag  bool
 	skipSetupFlag bool
+	coreCtxFlag   string
+	wds1CtxFlag   string
+	its1CtxFlag   string
+	wec1CtxFlag   string
+	wec2CtxFlag   string
 )
 
 func init() {
 	flag.BoolVar(&releasedFlag, "released", false, "released controls whether we use a release image")
 	flag.BoolVar(&skipSetupFlag, "skip-setup", false, "skip kubestellar cleanup and setup")
+	flag.StringVar(&coreCtxFlag, "core-ctx", "kind-kubeflex", "context for the KS core cluster")
```

Comment:
I think we generally call this the "kubeflex hosting cluster", or just "hosting cluster" for short.

Reply:
Fixed. thx!